ValeStyles/Couchbase/Latin.yml

```diff
@@ -0,0 +1,13 @@
+extends: substitution
+message: "Use '%s' instead of '%s'."
+link: https://docs.couchbase.com/styleguide/latin-abbreviations.html
+ignorecase: true
+level: error
+nonword: true
+action:
+  name: replace
+swap:
+  '(?i)\be*\.?g\.?\b': for example
+  '(?i)\bi\.?e\.?\b': specifically
+  '(?i)\betc\.?|etc\.?|e\.t\.c\b': and so on
```

Comment:
I think if you have alternatives `|` then you'll need to group them in parens, otherwise you'll get the `\b` only associated with the first and last alternative